successfactors-connector/processes/connector/EmpEmployment.p.json

```diff
@@ -0,0 +1,99 @@
+{
+  "format" : "10.0.0",
+  "id" : "191FB07A25556DA3",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "connector.EmpEmploymentData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "getEmpEmployment",
+      "config" : {
+        "callSignature" : "getEmpEmployment",
+        "input" : {
+          "params" : [
+            { "name" : "top", "type" : "Integer" },
+            { "name" : "skip", "type" : "Integer" },
+            { "name" : "filter", "type" : "String" },
+            { "name" : "count", "type" : "Boolean" },
+            { "name" : "orderBy", "type" : "java.util.List<String>" },
+            { "name" : "select", "type" : "java.util.List<String>" },
+            { "name" : "expand", "type" : "java.util.List<String>" },
+            { "name" : "id", "type" : "String" }
```

Comment:
Why do we need to add the id field? I don't see any usage for it. If it's not necessary, could you please remove it?